src/omega_edit/client.ts

```diff
@@ -97,25 +97,31 @@ export function activate(ctx: vscode.ExtensionContext) {
         randomId().toString(),
```

Comment:
I recommend leaving the desired session IDs undefined (unless they are explicitly named) to allow the server to generate them instead.  The C++ server will use a 36-character UUID and the Scala server will use an 8-character ID.  The servers make sure there are no collisions in the IDs.